OpenRA.Mods.Common/Traits/Player/LobbyPrerequisiteCheckbox.cs

```diff
@@ -49,7 +51,9 @@ public class LobbyPrerequisiteCheckboxInfo : TraitInfo, ILobbyOptions, ITechTree
 
 		IEnumerable<LobbyOption> ILobbyOptions.LobbyOptions(MapPreview map)
 		{
-			yield return new LobbyBooleanOption(ID, Label, Description,
+			yield return new LobbyBooleanOption(ID,
+				Game.ModData.Translation.GetString(Label),
+				Game.ModData.Translation.GetString(Description),
```

Comment:
Hm, since you're changing all LobbyOptions, can we move `Game.ModData.Translation.GetString` into the constructor of the base class to reduce duplication and make the calls a bit more slim?